### PR TITLE
[TA] Improve readme content

### PR DESCRIPTION
**Pull request by mssfang**
Review the REAMDE and  found some improvements

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 1]
```diff
@@ -20,6 +20,26 @@ and includes six main functions:
...
+```
+[//]: # ({x-version-update-end})
+**Note:** This version of the client library defaults to the `v3.1` version of the service.
+
+This table shows the relationship between SDK services and supported API versions of the service:
+
+|SDK version|Supported API version of service
+|-|-
+|5.0.x | 3.0
+|5.1.x | 3.0, 3.1 (default)
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/README.md:41]: maybe just me, but I feel like the more recent release should be at the top of the table.

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 2]
```diff
@@ -371,7 +379,7 @@ syncPoller.getFinalResult().forEach(
 ### Analyze multiple actions
 The `Analyze` functionality allows to choose which of the supported Text Analytics features to execute in the same
 set of documents. Currently, the supported features are: `entity recognition`, `linked entity recognition`,
-`key phrase extraction`, and `Personally Identifiable Information (PII) recognition`. 
+`Personally Identifiable Information (PII) entity recognition`, `key phrase extraction`, and `Sentiment analysis`.
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/README.md:382]: nit

```suggestion
`Personally Identifiable Information (PII) entity recognition`, `key phrase extraction`, and `sentiment analysis`. 
```

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 3]
```diff
@@ -20,6 +20,26 @@ and includes six main functions:
...
+</dependency>
+```
+[//]: # ({x-version-update-end})
+**Note:** This version of the client library defaults to the `v3.1` version of the service.
+
+This table shows the relationship between SDK services and supported API versions of the service:
+
+|SDK version|Supported API version of service
+|-|-
+|5.0.x | 3.0
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/README.md:40]: Do we need to mention the service version here?

Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:40]: I mimic the same pattern from Form Recognizer: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/README.md#include-the-package

Comment (samvaity) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:40]: Misunderstood the table, you're good.

**Review by samvaity — APPROVED** [review]

**Review by maririos — APPROVED** [review]

**Comment by mssfang** [comment]
The live test is been passed. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=985427&view=results
Double check with running one more time

**Pull request merged**